crates/bevy_asset/src/server/info.rs

```diff
@@ -189,7 +189,17 @@ impl AssetInfos {
         loading_mode: HandleLoadingMode,
         meta_transform: Option<MetaTransform>,
     ) -> Result<(UntypedHandle, bool), GetOrCreateHandleInternalError> {
-        match self.path_to_id.entry(path.clone()) {
+        // TODO: It is not currently possible to create or get a `Handle` without the `type_id` parameter.
+        // This function should
```

Comment:
TODO & suddenly ending sentence?

Should the behaviour here be to get any valid existing handle associated with the path?

Reply:
I've since resolved this TODO